src/Appwrite/Platform/Tasks/ScheduleMessages.php

```diff
@@ -59,7 +63,6 @@ protected function enqueueResources(Group $pools, Database $dbForConsole): void
 
                 $queueForDeletes
                     ->setType(DELETE_TYPE_SCHEDULES)
-                    ->setDocument($schedule)
```

Comment:
This delete job requires a datetime. And won't generating the same delete job after every message be excessive/inefficient?

Reply:
Resolved by deleting schedules directly here